src/components/JobsTrainingContent/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import "./index.scss";
+import { useTranslation } from "react-i18next";
+import OpportunitiesCard from "../OpportunitiesCard";
+import SearchBar from "../SearchBar";
+
+const JobsTrainingContent = () => {
+  const { t } = useTranslation();
+  const [translationKey, setTranslationKey] = useState("jobOpportunities");
+  const cardsArr = [...t(`home.${translationKey}`, { returnObjects: true })];
+  const cards = cardsArr.map((opportunity) => (
+    <OpportunitiesCard {...opportunity} key={opportunity.id} />
+  ));
+
+  const cardsList = cards.map((card, index) => {
+    return (
+      <Col
+        sm={12}
+        md={6}
+        className={index % 2 === 0 ? "peachBorder" : "greenBorder"}
```

Comment:
I have a note about changing cards color in each row, I think it would so nice if you can change the colors in each row not making each column with one color, for example:
```
peach green
green peach
peach green
.... ....
```
as I remember, our UI/UX supervisor gave us the same note about it.

Reply:
Actually, no the UI/UX designer wanted to keep all in one color. The thing you are suggesting is what we have in Figma right now. But I could not find away to do it. So, I did the PR like this for now because the functionality is more important. and I can discuss it with all of you.